dist/index.js

```diff
@@ -1168,6 +1168,13 @@ const installInOneFolder = ({ usePackageLock, workingDirectory }) => {
   })
 }
 
+const setAuthToken = async (authToken) => {
+  const npmPath = await io.which('npm', true)
+  const args = ['config', 'set', '//registry.npmjs.org/:_authToken', authToken]
```

Comment:
You are hard-coding the npm registry here, but it would be cool if any other private registry would be supported as well. Especially Github's, given that this is a Github action! 😄 

So another input `registry`, which defaults to the npm registry would make sense here I think.